keras_nlp/models/deberta_v3/deberta_v3_presets.py

```diff
@@ -25,7 +25,9 @@
             "max_sequence_length": 512,
             "bucket_size": 256,
         },
-        "preprocessor_config": {},
+        "preprocessor_config": {
+            "mask_token_id": 128000,
```

Comment:
I think we need to remove this from all the presets now right? It is breaking tests.